### ISSUE TITLE
Can't solve Calibration issue on bot Start

### ISSUE THREAD

**Issue by rotd0g** [steve1316/granblue-automation-pyautogui#164]
After successful install, Settings and Ready status i am receiving an error: 
  
  File "D:\GBF\backend\bot\game.py", line 921, in start_farming_mode
    Game._calibrate_game_window(display_info_check = True)
  File "D:\GBF\backend\bot\game.py", line 59, in _calibrate_game_window
    raise RuntimeError("Calibration of window dimensions failed. Is the Home button on the bottom bar visible?")
RuntimeError: Calibration of window dimensions failed. Is the Home button on the bottom bar visible?

My screen resolution is 2560x1440
Medium window size in game selected
i am in Home screen with Home button visible at the bottom right corner
i had 125% text in windows screen and reset it to 100%
Browser scaled to 100% as well
Tried with both Chrome and Edge

Hope you can help, thanks in advance!

**Comment by steve1316** [comment]
Hmm, can you provide a fullscreen screenshot of what the game window looks like? You can either attach it here or through discord.

**Comment by rotd0g** [comment]
@steve1316 wasn't able to locate your discord, posting here
![source](https://user-images.githubusercontent.com/49983155/219617233-0bac9110-6083-4d44-8a9b-c81ffc6c2227.png)

**Comment by steve1316** [comment]
Can you turn on `Debug Mode` in the Settings page and post the full logs here?

**Comment by rotd0g** [comment]
Hi Steve, 
Logs attached: [log 2023-02-18 10-42-19.txt](https://github.com/steve1316/granblue-automation-pyautogui/files/10773622/log.2023-02-18.10-42-19.txt)

We investigated yesterday with a fellow gbf player and found out the following:
Chrome colors, gamma and size sems to be affected by some other application(possibly NVidia but we found no evidence in it's Control Panel/Settings) even with default graphics and size setup chrome seems to be scaled to 110% and darker than it should be by default. 
Key to this comparison was Portable Custom Chromium browser which is not affected by 'this'
I checked my Chrome Extensions, Chrome UI Settings and there's nothing.
I'll try to investigate what else could be causing this issue.

**Comment by Vanishborn** [comment]
Hi! Sorry to jump into the discussion like this, but I've encountered the same issue. Attached are the logs with debug on. I tried on both Edge and Chrome. My screen resolution is 3840x2160 with scale of 150x. The bot only works when I turn off scale (back to 100%) which makes everything TINY. 2 log files are before and after with respect to time created.

Hope this helps. And although changing my screen scale every time I run the game is not that big of an issue for me, I do wish the program would work without all this hassle. Anyway, thanks for this amazing tool.

[log 2023-04-8 08-23-03.txt](https://github.com/steve1316/granblue-automation-pyautogui/files/11183579/log.2023-04-8.08-23-03.txt)
[log 2023-04-8 08-26-14.txt](https://github.com/steve1316/granblue-automation-pyautogui/files/11183580/log.2023-04-8.08-26-14.txt)
